glui_clipboard.cpp

```diff
@@ -0,0 +1,303 @@
+/****************************************************************************
+
+  GLUI User Interface Toolkit
+  ---------------------------
+
+     glui_clipboard_x11.cpp - GLUI_Clipboard_Object class
+
+	 This file -is- in the public domain. Circa summer 2017.
```

Comment:
Actually, I wasn't sure what the GLUI license situation even is. So I looked and noticed the "This program is -not- in the public domain." part, and I think what I wrote, that you take issue with, was just a joke in reference to that. And that it made me feel a little uncomfortable to attach myself too.